rosa_tutorials/rosa-mobb-aws-cloudwatch-publish-metrics.adoc

```diff
@@ -0,0 +1,177 @@
+:_content-type: ASSEMBLY
+[id="rosa-mobb-aws-cloudwatch-publish-metrics"]
+= Tutorial: Using the AWS CloudWatch agent to publish metrics to CloudWatch in ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-aws-cloudwatch-publish-metrics
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-19
+//---
+//date: '2021-10-04'
+//title: Using the AWS Cloud Watch agent to publish metrics to CloudWatch in ROSA
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Kevin Collins
+//---
+
+You can use the AWS CloudWatch agent to scrape Prometheus endpoints and publish metrics to CloudWatch in a {product-title} (ROSA) cluster.
+
+It pulls from The AWS documentation for installing the CloudWatch agent to Kubernetes and collections and publishes metrics for the Kubernetes API Server and provides a simple Dashboard to view the results.
+
+[IMPORTANT]
+====
+The AWS CloudWatch Agent does link:https://github.com/aws/amazon-cloudwatch-agent/issues/187[not support] pulling all metrics from the Prometheus federated endpoint.
+====
+
+.Prerequisites
+
+* The link:https://aws.amazon.com/cli/[AWS CLI]
+* The link:https://stedolan.github.io/jq/[`jq` command]
+* A ROSA cluster
```

Comment:
There should be another Prerequisite that a user with group cluster-admins created.
I tried, dedicated-admin cannot work